### DISCUSSION TITLE
Why kernelsu don't contains gapps?

### DISCUSSION THREAD

**Discussion by GitHub-Creeper** [LSPosed/MagiskOnWSALocal#789] [Q&A] [answered]
Why when I select kernelsu root, i don't find gapps select option anywhere, and so do the command, i don't any ''gapps'' word in the select menu and in the installation process, do it already have gapps or kernelsu don't support it?

**Comment by Doddddd** [comment, marked as answer]
https://github.com/LSPosed/MagiskOnWSALocal/blob/main/scripts/build.sh#L307

**Reply by GitHub-Creeper** [reply]
Oh I understand. GApps forces you to install Magisk and KernelSU and GApps is unsupported. What a shame but okay I'll use Magisk instead

**Comment by Howard20181** [comment]
Wait for Magisk for kiwi (HPE).
